L78-python-rich-server-context.md

```diff
@@ -0,0 +1,81 @@
+Title
+----
+* Author(s): stpierre
+* Approver: gnossen
+* Status: Draft
+* Implemented in: [python](https://github.com/grpc/grpc/pull/25457)
+* Last updated: 2021-02-17
+* Discussion at: https://groups.google.com/g/grpc-io/c/ctZJkd4MfHg
+
+## Abstract
+
+Expose context status code, details, and trailing metadata on the
+server side.
+
+## Background
+
+Currently, on the server-side `ServicerContext` object, status code,
+details, and trailing metadata are private. As a result, an
+interceptor cannot take action based on any of those properties.
+
+## Proposal
+
+We will add three methods to the servicer context interface
+(`grpc.ServicerContext`), and synchronous (`grpc._server._Context`)
+and AIO server (`grpc.aio.ServicerContext`) implementations:
```

Comment:
fyi. there is a PR about adding this to asyncio https://github.com/grpc/grpc/pull/25600, but I haven't heard back for a while.